sled-agent/src/services.rs

```diff
@@ -2289,114 +2290,298 @@ impl ServiceManager {
+                                        "wicketd can only be started in the ",
+                                        "switch zone",
+                                    )
+                                    .to_string(),
+                                });
+                            };
+
+                            let rack_subnet = Ipv6Subnet::<AZ_PREFIX>::new(
+                                info.underlay_address,
+                            );
+
+                            let baseboard_info = serde_json::to_string_pretty(&baseboard)?;
+
+                            let wicketd_config =
+                                PropertyGroupBuilder::new("config")
+                                    .add_property(
+                                        "address",
+                                        "astring",
+                                        &format!("[::1]:{WICKETD_PORT}"),
+                                    )
+                                    .add_property(
+                                        "artifact-address",
+                                        "astring",
+                                        &format!("[{bootstrap_address}]:{BOOTSTRAP_ARTIFACT_PORT}"),
+                                    )
+                                    .add_property(
+                                        "mgs-address",
+                                        "astring",
+                                        &format!("[::1]:{MGS_PORT}"),
+                                    )
+                                    // We intentionally bind `nexus-proxy-address` to
+                                    // `::` so wicketd will serve this on all
+                                    // interfaces, particularly the tech port
+                                    // interfaces, allowing external clients to connect
+                                    // to this Nexus proxy.
+                                    .add_property(
+                                        "nexus-proxy-address",
+                                        "astring",
+                                        &format!("[::]:{WICKETD_NEXUS_PROXY_PORT}"),
+                                    )
+                                    .add_property(
+                                        "rack-subnet",
+                                        "astring",
+                                        &rack_subnet.net().ip().to_string(),
+                                    )
+                                    .add_property(
+                                        "config/baseboard-file",
+                                        "astring",
+                                        // TODO: Used to be "/opt/oxide/baseboard.json", make sure
+                                        // it's OK to change
+                                        "/var/svc/manifest/site/wicketd/baseboard.json",
+                                    )
```

Comment:
I'd like to change the location of this file since we know it's relevant to wicket only. Just want to double check I'm not breaking anything here.